### PR TITLE
Fixes #21471: 

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/21471

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/rudder.js [hunk 1]
```diff
@@ -37,6 +37,22 @@
...
+  '<': '&lt;',
+  '>': '&gt;',
+  '"': '&quot;',
+  "'": '&#39;',
+  '/': '&#x2F;',
+  '`': '&#x60;',
+  '=': '&#x3D;'
+};
+
+function escapeHTML (string) {
```
Comment (ncharles) [inline webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/rudder.js:51]: could you also remove safeTagsReplace with that (cf https://github.com/Normation/rudder/pull/4390/files )

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR updated with a new commit

**Review by ncharles — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**